From e5c980a7e132dd987040a11775198595dd58b48d Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 14 Mar 2019 03:55:51 -0700
Subject: [PATCH 0565/1852] dt-bindings: dma: xilinx_frmbuf: Make max-width and
 max-height as mandatory properties

commit 7b8d78fdb21b912fd675f345ed3a1073ac0fe9cf from
https://github.com/Xilinx/linux-xlnx.git

This patch makes xlnx,max-width and xlnx,max-height as mandatory
properties. This breaks backward compatibility.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt      | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt b/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
index 39cb6ff762c2..aa068f68a8c0 100644
--- a/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
+++ b/Documentation/devicetree/bindings/dma/xilinx/xilinx_frmbuf.txt
@@ -36,6 +36,10 @@ Required Properties Common to both FB Read and FB Write for v2.1:
 - xlnx,pixels-per-clock	: Pixels per clock set in IP (1, 2 or 4)
 - clocks: Reference to the AXI Streaming clock feeding the AP_CLK
 - clock-names: Must have "ap_clk"
+- xlnx,max-height	: Maximum number of lines.
+			  Valid range from 64 to 4320.
+- xlnx,max-width	: Maximum number of pixels in a line.
+			  Valid range from 64 to 8192.
 
 Optional Properties Common to both FB Read and FB Write for v2.1:
 - xlnx,dma-align	: DMA alignment required in bytes.
@@ -48,10 +52,6 @@ Optional Properties Common to both FB Read and FB Write for v2.1:
 - xlnx,fid		: Field ID enabled for interlaced video support.
 			  Can be absent for progressive video.
 
-Optional properties:
-- xlnx,max-height	: Maximum number of lines.
-- xlnx,max-width	: Maximum number of pixels in a line.
-
 VIDEO FORMATS
 The following table describes the legal string values to be used for
 the xlnx,vid-formats property.  To the left is the string value and the
-- 
2.31.1

